fluss-server/src/main/java/com/alibaba/fluss/server/coordinator/CoordinatorService.java

```diff
@@ -197,6 +203,49 @@ public CompletableFuture<DropTableResponse> dropTable(DropTableRequest request)
         return CompletableFuture.completedFuture(response);
     }
 
+    @Override
+    public CompletableFuture<AddPartitionResponse> addPartition(AddPartitionRequest request) {
+        AddPartitionResponse response = new AddPartitionResponse();
+        TablePath tablePath = toTablePath(request.getTablePath());
+        if (!metadataManager.tableExists(tablePath)) {
```

Comment:
we can remove this check, as the `metadataManager#getTable` will also check the existence.

Reply:
Done!